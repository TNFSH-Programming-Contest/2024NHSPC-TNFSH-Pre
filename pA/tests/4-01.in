728010920840585595 48164038647779127 745862414496274333
425360106025462275 480744840199233335 615932427759943445
