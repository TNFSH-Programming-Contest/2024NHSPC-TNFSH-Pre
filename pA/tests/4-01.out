879723732309914909
